### PR TITLE
Move from toolkit

### PR DESCRIPTION
**Pull request by ZachEddy**
# Move from Atlas toolkit

This pull request moves the `protoc-gen-perm` plugin from the Atlas toolkit to its own repository in infobloxopen.

Note to reviewers: the commits following 1aafb11 are my own changes. You don't have to review the entire pull request (feel free to do so), but please check the additional commits that I've added. I kept the individual commits pretty small.

### CODE REVIEW

**Comment by ZachEddy** [comment]
Hey @Evgeniy-L and @dkukharau, any ideas why the build is failing? The tests pass locally, but not on Travis CI 🤔

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+GOPATH ?= $(HOME)/go
+SRCPATH := $(patsubst %/,%,$(GOPATH))/src
+
+default: options install
+
+
+.PHONY: options
+options:
+	protoc -I. -I$(SRCPATH) -I../../vendor  \
```
Comment (ystankevich) [inline Makefile]: Need to replace "../../vendor" on "./vendor" and on line 19 too.

**Comment by ystankevich** [comment]
Build may fail, because protoc_test.go uses "protoc" for generating code, but travis file doesn't install it.

**Comment by ZachEddy** [comment]
@ystankevich The `protoc` command isn't called in Travis. Rather than generate the protobuf files in Travis, I just run `go test -v ./...` to run tests against the existing protobuf files. This is just a temporary measure because the Atlas gentool doesn't have `protoc-gen-perm` yet.

For some mysterious reason, the tests are failing on Travis, but they do not fail in my development environment (even though my working tree is clean).

Edit: never mind. I see the issue now. Thanks for the input!

**Review by ystankevich — APPROVED** [review]

**Pull request merged**